### PR TITLE
MIN_BY, MAX_BY, ARBITRARY, KURTOSIS & SKEWNESS Aggregate Functions

### PR DESCRIPTION
**Pull request by pdet**
This PR adds the following statistical functions:
* min_by 
* max_by
* arbitrary
* kurtosis
* skewness

### CODE REVIEW

File: test/sql/aggregate/aggregates/test_skewness.test [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# name: test/sql/aggregate/aggregates/test_skewness.test
+# description: Test skewness aggregate
+# group: [aggregates]
+
+##Corner cases
+#statement error
```
Comment (Mytherin) [inline test/sql/aggregate/aggregates/test_skewness.test:6]: Is this commented out by accident?

**Review by Mytherin — COMMENTED** [review]
Thanks for the PR. Looks good. One minor comment and then it can be merged, minus some clang-tidy fixes after the merge:

File: test/sql/aggregate/aggregates/test_skewness.test (hunk 1 above)
Comment (pdet) [reply test/sql/aggregate/aggregates/test_skewness.test:6]: jup
